api/types/types.proto

```diff
@@ -269,6 +272,8 @@ message DatabaseStatusV3 {
     string CACert = 1 [ (gogoproto.jsontag) = "ca_cert,omitempty" ];
     // AWS is the auto-discovered AWS cloud database metadata.
     AWS AWS = 2 [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "aws" ];
+    // MySQLServerVersion is the server version read from a MySQL server handshake message.
+    string MySQLServerVersion = 3 [ (gogoproto.jsontag) = "mysql_server_version,omitempty" ];
```

Comment:
Here too, can we just `mysql.server_version` here as well? Reuse the same `message MySQLOptions` struct basically.

Reply:
Changed